src/index.js

```diff
@@ -138,10 +145,16 @@ function mdReactFactory(options={}) {
       assign(tree.shift(), { key }) :
       { key };
 
-    if (level === 0 && className) {
-      props.className = className;
+    if (level === 0) {
+      if (className) {
+        props.className = className;
+      }
+      if (style) {
+        props.style = style;
+      }
     }
```

Comment:
I think it's ok to proxy all the rest of the props to root component, like this:
```js
<...>, ...rest } = options;
let props = <...>
if (level === 0) {
  props = { ...props, ...rest };
}
```